### PR TITLE
feat(website): update website index styles

### PR DESCRIPTION
**Pull request by Carlosfengv**
Adjust the home page style, including the cookie prompt, and adapt it to the mobile end.

I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Summary about this PR

Closes #issue

### CODE REVIEW

**Comment by Carlosfengv** [comment]
@Chasen-Zhang

**Comment by Chasen-Zhang** [comment]
image

The mobile style is a little messy @Carlosfengv

File: website/src/css/font.scss [hunk 1]
```diff
@@ -0,0 +1,109 @@
+// @font-face {
```
Comment (Xuanwo) [inline website/src/css/font.scss:1]: **request**: Pleaes remove not needed code.

File: website/src/pages/index.tsx [hunk 2]
```diff
@@ -1,32 +1,84 @@
...
+  },
+  {
+    'icon':'Twitter',
+    'title': 'Follow our twitter',
+    'link': 'https://twitter.com/Datafuse_Labs'
+  },
+  {
+    'icon':'Youtube',
+    'title': 'Youtube',
+    'link': 'https://www.youtube.com/@databend8089'
```
Comment (Xuanwo) [inline website/src/pages/index.tsx:29]: **Idea**: We really need a better channel name.

cc @wubx & @PsiACE, `@databend8089` is so weird.

File: website/src/css/font.scss (hunk 1 above)
Comment (Carlosfengv) [reply website/src/css/font.scss]: 👌

File: website/src/pages/index.tsx (hunk 2 above)
Comment (BohuTANG) [reply website/src/pages/index.tsx:29]: We will change to new in another PR.

**Review by Chasen-Zhang — APPROVED** [review]

**Comment by wubx** [comment]
Hold, Let's change youtube address.

**Review by wubx — APPROVED** [review]

**Comment by BohuTANG** [comment]
Wait @soyeric128 check all the statements.

**Comment by soyeric128** [comment]
Because you have the title "Join our growing community", you dont need **Follow** or **Join** again when you list the social media down there. Make sure you you spell the product names EXACTLY as their formal ones.

1. Youtube -> YouTube

youtube

2. Github -> GitHub

Screenshot 2023-02-23 at 9 25 02 AM

3. Follow our twitter -> Twitter
4. Join our slack channel -> Slack


Opensource quickstart -> Tutorials

🎉 Databend Cloud is now available. -> Databend Cloud Now Available


No Operations Required, Ready-to-Use. Register Now and Get $200 Discount. -> What **Operations** not required ?, Installation ???** @BohuTANG

**Comment by soyeric128** [comment]
Ready to use out of the box. Register now and get a $200 coupon.

**Review by wubx — APPROVED** [review]

File: website/src/pages/index.tsx [hunk 3]
```diff
@@ -8,9 +8,9 @@ import * as icons from "../components/Icons"
 
 const community = [
   {
-    'icon':'Github',
+    'icon':'Gitub',
```
[the remark this answers is not in the document]
Comment (Carlosfengv) [reply website/src/pages/index.tsx]: It seems to be a typo of commit 89b8874, which has been fixed in commit 705bf8e.

**Pull request merged**